### DISCUSSION TITLE
Creating aliases or word shortcuts for specific softwares used often

### DISCUSSION THREAD

**Discussion by LBF38** [Flow-Launcher/Flow.Launcher#1455] [Ideas]
This idea came when I use a same software regularly. Therefore, we can put "vs code" for example, for launching "Visual Studio Code".

What do you think of this feature ?

**Comment by taooceros** [comment]
already in plan. see #930

**Reply by LBF38** [reply]
thanks for the reply ! Excited to get hands on it !

**Comment by deefrawley** [comment]
That's what the Plugin Runner plugin does, only you need to set your shortcuts up manually.

**Reply by LBF38** [reply]
Thank you, I'll check the plugin
